.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+  schedule:
+    - cron: '28 11 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        language: [ 'javascript' ]
```

Comment:
Shouldn't this be `typescript`

Reply:
👍 Great catch, I'll make sure that gets updated.  Thanks!

Reply:
Is there actually a difference in this? My assumption was that `javascript` did cover TypeScript even though it is listed in the docs as a different language.

I'm now wondering if a bunch of my repos might have been getting only half-checked because of this 😅

Reply:
Yeah after reading through the docs the best strategy seemed to be to use both for any repo where the source had both TypeScript and JavaScript i.e.:

`language: [ 'javascript', 'typescript' ]` < this seems to be working correctly for mixed source.